README.md

```diff
@@ -53,7 +61,19 @@ describe input("some_input") do
 end
 ```
 
-Chef InSpec will determine a secret lookup path and access Vault. With no other settings, Chef InSpec will look for a Vault secret located at `secret/inspec/my_profile` with a key named "some_input". Chef InSpec will use the Vault secret if found, but otherwise it will fall back to other means of resolving the input, such as the profile metadata or CLI values.
+With no other settings, Chef InSpec will look for a Vault secret located at `secret/inspec/my_profile` with a key named `some_input`. The `my_profile` part of the path is the name of your InSpec profile. The `inspec` part comes from the `path_prefix` setting.
+
+### Absolute Path Lookup
+
+In some cases, you want to access global information and not something related to a specific profile. For this case, qualify the path in absolute syntax with a starting `/`, like this:
```

Comment:
```suggestion
To access global information instead of a value related to a specific profile, qualify the path in absolute syntax with a starting `/`. For example:
```